code/modules/mob/living/carbon/human/human.dm

```diff
@@ -231,11 +231,12 @@
 
 /mob/living/carbon/human/Topic(href, href_list)
 	if(usr.canUseTopic(src, BE_CLOSE, NO_DEXTERY))
-
 		if(href_list["embedded_object"])
-			var/obj/item/I = locate(href_list["embedded_object"])
-			var/obj/item/bodypart/L = locate(href_list["embedded_limb"])
-			if(!I || !L || I.loc != src || !(I in L.embedded_objects)) //no item, no limb, or item is not in limb or in the person anymore

```

Comment:
didn't say that.
just that you're a babbu.

(also it's just as safe as the rest of hrefs (read: none of it is safe))